client/src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Button({ onDelete, id }) {
```

Comment:
A couple of things to think about with this component:
1. "Button" is a very general name - can you think of a more specific name for what the button actually is for?
2. `onDelete` is a slightly strange name - "on" as a prefix normally means this function is meant to be called when something has happened (e.g. onClick is for when something has been clicked), but this actually _does_ the delete, rather than handles it. Can you think of a name which suggests more this _does_ the delete, rather than is called as a _result_ of the delete?

Reply:
Thank you @illicitonion for pointing these out and paying attention to details! Having the right names is super helpful for me to understand my own code when I look at them.
After going through your review, I made changes to my level 300 as my level 300 is built on top of my level 250. 
1. I have renamed my Button component to DeleteButton.
2. onDelete does sound strange. I have renamed it to handleDelete.